tests/browser/recurring_donations/create_recurring_donation.robot

```diff
@@ -2,17 +2,15 @@
 
 Resource        tests/NPSP.robot
 Suite Setup     Open Test Browser
-Suite Teardown  Delete Records and Close Browser
+#Suite Teardown  Delete Records and Close Browser
 
 
 *** Test Cases ***
 
 Create Open Recurring Donation With Monthly Installment
     [tags]  unstable
-    Sleep    5
-    Open App Launcher
-    Populate Address    Search apps or items...    NPSP Settings
-    Select App Launcher Link  NPSP Settings
+    Select App Launcher Tab  NPSP Settings
+    Sleep    3
```

Comment:
This sleep should not be needed since it's followed by waiting for a specific locator.

Reply:
was done while testing